src/etc/inc/captiveportal.inc

```diff
@@ -347,15 +347,10 @@ EOD;
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-error.html");
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-logout.html");
 
-		captiveportal_radius_stop_all();
+		captiveportal_disconnect_all(10); // NAS-Request
 
 		captiveportal_filterdns_configure();
 
-		/* send Accounting-Off to server */
-		if (!platform_booting()) {
-			captiveportal_send_server_accounting(true);
```

Comment:
There's not much we can test, basically only "is this zone the only active zone using RADIUS accounting with this specific RADIUS server?" and even if the answer is yes, by that time we've already sent Accounting-Stop packets for all the clients so Accounting-Off buys us nothing. 
Given that Accounting-Off is optional I'd just not bother sending it, maybe only when rebooting, after all clients have been cleanly disconnected.